code/modules/mining/ores_coins.dm

```diff
@@ -131,6 +131,10 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 	icon_state = "volcanic_sand"
 	singular_name = "volcanic ash pile"
 
+/obj/item/stack/ore/glass/basalt/Initialize(mapload, new_amount, merge = TRUE)
+	recipes = GLOB.sand_recipes
+	. = ..()
```

Comment:
Basalt is already a subtype of glass, so its Initialize() already does recipes = GLOB.sand_recipes. Calling ..() here will just make it do so twice. In other words: this PR does nothing. Try to figure out the problem in particular.